hydrogen-vm-render-script.js

```diff
@@ -19,18 +19,111 @@ const {
   setupLightboxNavigation,
   ObservableArray,
   TimelineViewModel,
-  TimelineView
+  TimelineView,
+  createMemoryStorage,
+  MockClock,
+  Client,
+  ConsoleLogger,
+  LoadStatus,
+  SyncStatus,
 } = require('hydrogen-view-sdk');
 
 const events = require('./sample-test-messages.json').chunk;
 
-
 // eslint-disable-next-line max-statements
-function mountHydrogen() {
+async function mountHydrogen() {
+  let hasSynced = false;
+  const platform = {
+    clock: new MockClock(),
+    random: Math.random,
+    loadOlm() {},
+    loadOlmWorker() {},
+    storageFactory: {
+      create(sessionId, logItem) {
+        return createMemoryStorage();
+      },
+    },
+    sessionInfoStorage: {
+      add() {},
+    },
+    request(url, requestOptions) {
+      console.log('request url', url);
+      return {
+        abort() {},
+        async response() {
+          if (url.endsWith('/sync?timeout=0&filter=1')) {
+            if (hasSynced) {
+              return {
+                status: 200,
+                body: {},
+              };
+            } else {
+              hasSynced = true;
+              return {
+                status: 200,
+                body: {
+                  rooms: {
+                    join: {
+                      '!awef:awef.com': {
+                        timeline: {
+                          events: events,
+                        },
+                      },
+                    },
+                  },
+                },
+              };
+            }
+          }
+
+          if (!hasSynced) {
+            if (url.endsWith('/filter?')) {
+              return {
+                status: 200,
+                body: {
+                  filter_id: '1',
+                },
+              };
+            } else if (url.endsWith('/versions?')) {
+              return {
+                status: 200,
+                body: {
+                  versions: [],
+                },
+              };
+            }
+          } else {
+            // Stall the other requests
+            return new Promise((resolve) => {});
+          }
+
+          return { status: 404, body: 'Stub not implemented' };
+        },
+      };
+    },
+  };
+
+  const logger = new ConsoleLogger({ platform });
+  platform.logger = logger;
+
+  console.log('Client', Client);
+  const client = new Client(platform);
+  client.startWithAuthData({
+    accessToken: 'fake',
+    deviceId: 'fakedeviceId',
+    userId: 'fakeUserId',
+    homeserver: 'fakehomeserverUrl',
+  });
+  await client.loadStatus.waitFor((status) => {
```

Comment:
actually, don't think this is really needed given we wait for the first sync to complete already below